pom.xml

```diff
@@ -902,6 +908,28 @@
             <autoVersionSubmodules>true</autoVersionSubmodules>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>findbugs-maven-plugin</artifactId>
+          <version>${findbugs.version}</version>
+          <dependencies>
+            <dependency>
+              <groupId>org.apache.beam</groupId>
+              <artifactId>java-build-tools</artifactId>
+              <version>${project.version}</version>
+            </dependency>
+          </dependencies>
+          <configuration>
+            <excludeFilterFile>beam/findbugs-filter.xml</excludeFilterFile>
+          </configuration>
+          <executions>
+            <execution>
+              <goals>
+                <goal>check</goal>
```

Comment:
Added explicit phase. Earlier phases to consider would be `compile`, `test`, `package`. I don't see any reason it couldn't execute in `test`, but I'm not sure. I'll try it both ways and verify that I get the same results.